tests/integration/helpers.bash

```diff
@@ -3,25 +3,15 @@
 # Root directory of integration tests.
 INTEGRATION_ROOT=$(dirname "$(readlink -f "$BASH_SOURCE")")
 
-. ${INTEGRATION_ROOT}/multi-arch.bash
+# Download images, get *_IMAGE variables.
+eval $("${INTEGRATION_ROOT}"/get-images.sh)
```

Comment:
`eval` on this output seems kind of weird - why isn't this sourced instead?
(if there's a good reason for doing so, perhaps a comment for posterity would make sense :innocent:)

Additionally, if `get-images.sh` fails, this likely won't fail properly (you'd want something like `GET_IMAGES="$(.../get-images.sh)"; eval "$GET_IMAGES"` instead). :eyes:

Reply:
This script is also used from two other places (`libcontainer/integration/utils_test.go` and `tests/rootless.sh`) and thus I want it to be executed in the same manner from all the places, thus the eval.

Sourcing it would have a few side effects, like
 - setting shell options (`set -e -u -o pipefail`) -- fix for that is kinda ugly as it requires to use another sub-shell inside the script;
 - setting additional env vars -- fix would be to add `unset XXX` to the end of the script, but when we'll have to maintain that list for unset.
 
Besides, if we source it, someone would spot the same variables (`$TESTDATA` and `$INTEGRATION_ROOT`) being used and will try to optimize it out, which is wrong.

Anyway, it all boils down to -- this is a standalone script. I have emphasized it in the comment at the very beginning.

> Additionally, if get-images.sh fails, this likely won't fail properly (you'd want something like GET_IMAGES="$(.../get-images.sh)"; eval "$GET_IMAGES" instead). ￼

￼Good catch; fixed!

PTAL